gradebook/app/ui/src/bundle/org/sakaiproject/tool/gradebook/bundle/Messages_en_GB.properties

```diff
@@ -1,210 +1,126 @@
-feedback_options_grade_display_assignment_grades=Display any released markbook items and entered scores to students. Markbook items that have been individually set to "released" will not display to students until this is checked.
+course_grade_details_null_msg=Currently, the gradebook does not include gradebook items that have not been marked in the course mark calculation. This could result in a higher course mark for a student. To include a gradebook item in the course mark calculation, you must enter a mark of zero or higher.  If you want to enter zeroes for any gradebook item that has not been marked and then re-calculate the course mark, click Calculate Course Marks. 
 grader_rules_select_a_grader=Select a marker to edit\:
-import_assignment_entire_notsupported=At least one markbook item in your upload has a non-numeric score. This markbook is not currently set up to accept non-numeric scores.
 inst_view_page_title=Marks for {0}
 feedback_options_grade_type=Mark Type
 feedback_options_grade_header=Mark
-gb_setup_title=Markbook Setup
-assignment_details_remove=Remove markbook item from markbook
 inst_view_log_title=Mark Log\: {0}
-import_entire_invalid_letter=At least one markbook item in your upload has an invalid letter mark.
+import_entire_invalid_letter=At least one gradebook item in your upload has an invalid letter mark.
 grade_scale_grade=Mark
-import_assignment_header=Markbook Item
 grader_rules_no_rules=There are currently no rules defined for this marker.
 export_grade_for_sis_prefix=course_mark_sis
-add_assignment_page_title=Add Markbook Item(s)
-edit_assignment_instruction=Submit this form to edit a markbook item in the markbook.
-remove_assignment_success=Markbook item {0} has been deleted.
-student_view_assignments=Markbook Items
-loading_dock_instructions=Below is the loading dock, a holding place for spreadsheets (csv format) that have been uploaded from your local computer. <br> Once loaded here, the data in the spreadsheet can be imported in part or whole into your markbook.
-assignment_details_page_title=Markbook Item Summary
+loading_dock_instructions=Below is the loading dock, a holding place for spreadsheets (csv format) that have been uploaded from your local computer. <br> Once loaded here, the data in the spreadsheet can be imported in part or whole into your gradebook.
 gb_setup_no_grade_entry_scale=No mark entry scale for letter marking has been defined.
-import_entire_some_success=Some markbook items were imported successfully. See below for the errors that did occur.  
 feedback_options_cannot_change_percentage2=Campus Teaching Centre
-feedback_options_cannot_change_percentage1=Alert\: Due to problems with this markbook configuration your changes have not been saved. Please contact your 
-roster_export_csv=Export Markbook
+roster_export_csv=Export Gradebook

```

Comment:
This line can go I think as it doesn't reference marks.

Reply:
The default property is 'Export as CSV' so I made the change above.

Reply:
Ok.